source/iterators.tex

```diff
@@ -1688,16 +1688,23 @@
     DerivedFrom<@\placeholdernc{ITER_CONCEPT}@(I), contiguous_iterator_tag> &&
     is_lvalue_reference_v<iter_reference_t<I>> &&
     Same<iter_value_t<I>, remove_cvref_t<iter_reference_t<I>>>;
```

Comment:
The semicolon at the end should be replaced by "&&".

Reply:
yup - missed that.  thanks